packages/quantic/force-app/main/default/lwc/quanticPager/e2e/fixture.ts

```diff
@@ -0,0 +1,28 @@
+import {PagerObject} from './page-object';
+import {quanticBase} from '../../../../../../playwright/fixtures/base-fixture';
+
+import {SearchObject} from '../../../../../../playwright/page-object/search-object';
+
+type QuanticPagerE2EFixtures = {
+  pager: PagerObject;
+};
+
+export const testSearch = quanticBase.extend<QuanticPagerE2EFixtures>({
```

Comment:
Why do you need two fixtures?

`search` and `insight` are already defined on the [`quanticBase` fixture](packages/quantic/playwright/fixtures/base-fixture.ts), so I think you should only have to extend it once to define `pager`.

Reply:
Actually I think the names I choose for these fixtures are causing confusions, 

I did some updates that are now making the structure in the following way:

1- the **quanticBase** fixture now contains two things:

- the **configuration** object: responsible of configuring the Quantic Example page to initializes the test environment for a given test.
- the **insightSetup** object: responsible of the extra setup step to initialize a quantic insight interface.

These two objects will. pretty much be needed by every Quantic  in the search and insight use case, that's why they are part of the **quanticBase** fixture 

2- the pager contains two fixtures:`testSearch` and `testInsight` for the search use case and the insight use case respectively.
Each fixture of these two append the `pager` object and the `search` object.

The reason why we need two fixtures is because each fixture is using the `search` object in a different way:

The `search` object is responsible of performing a search and intercepting the search requests.

- the `testSearch` fixture is using the `search` object with the parameter `useCase` set to the value `search` : `new SearchObject(page, 'search')` which will make the `search` object intercept/mock the urls that are used in the search use case: `/\/rest\/search\/v2\?organizationId=.*/`
-  the `testinsight` fixture is using the `search` object with the parameter `useCase` set to the value `insight` : `new SearchObject(page, 'insight')` which will make the `search` object intercept/mock the urls that are used in the insight use case: `/\/rest\/organizations\/.*\/insight\/v1\/configs\/.*\/search$/`